### ISSUE TITLE
Make Account Menu Usable With Keyboard Input

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1688]
Currently, the account menu is not usable with keyboard input. Focus should be on the first element of the menu when it's opened, you should be able to navigate it with a keyboard and information should be read out by a screen reader.

<img width="257" height="216" alt="Image" src="https://github.com/user-attachments/assets/072b9edd-1d1c-4ab7-b30f-5bfa6c34471e" />
